subprojects/maven/src/integTest/groovy/org/gradle/api/publish/maven/MavenPublishVersionRangeIntegTest.groovy

```diff
@@ -51,6 +59,14 @@ class MavenPublishVersionRangeIntegTest extends AbstractMavenPublishIntegTest {
                 api "group:projectD:[1.0,2.0)"
                 api "group:projectE:[1.0]"
             }"""
+        if (GradleContextualExecuter.configCache) {
```

Comment:
Copied from https://github.com/gradle/gradle/blob/52857ba4ed4cc3f31b3c9f92ef0f2a9eb45f4faf/subprojects/maven/src/integTest/groovy/org/gradle/api/publish/maven/AbstractMavenPublishJavaIntegTest.groovy#L87

Questions:

1. why not always declare the repo (with or without CC)?
2. same for dependencies - they may not be needed when running without CC, but is that something we want to guarantee? Isn't CC the intended mode of operation going forward, and all tests will need to be changed to be setup so dependency resolution during configuration works?

Reply:
My take would be to always have a sane dependency setup instead of a minimal one that fits the current needs of publishing.
The fact that these needs are changing with trying to make it CC compatible is a good indication that we should have tests independent of that.
None of these tests would work if we tried to compile something or run a test, so they are crippled in that sense.

So +1 from me to make them "complete" by having declared repositories and required dependencies published.